sway-ir/src/optimize/memcpyopt.rs

```diff
@@ -1025,3 +1053,255 @@ fn load_store_to_memcopy(context: &mut Context, function: Function) -> Result<bo
 
     Ok(true)
 }
+
+pub const MEMCPYPROP_REVERSE_NAME: &str = "memcpyprop_reverse";
+
+pub fn create_memcpyprop_reverse_pass() -> Pass {
+    Pass {
+        name: MEMCPYPROP_REVERSE_NAME,
+        descr: "Memcpyprop Reverse: Copy propagation of memcpy instructions",
```

Comment:
To make it consistent with other descriptions.
```suggestion
        descr: "Copy propagation of MemCopy instructions",
```